public/app/features/variables/custom/CustomVariableEditor.tsx

```diff
@@ -44,8 +56,24 @@ export class CustomVariableEditor extends PureComponent<Props> {
             />
           </div>
         </div>
-        <SelectionOptionsEditor variable={this.props.variable} onPropChange={this.onSelectionOptionsChange} />
+        <SelectionOptionsEditor
+          variable={this.props.variable}
+          onPropChange={this.onSelectionOptionsChange}
+          onMultiChanged={this.props.changeVariableMultiValue}
+        />
```

Comment:
I just realized something couldn't we have the VariableEditorEditor handle the dispatching of `changeVariableMultiValue`? Then these components would not have to connect to state. At the same time, maybe we want this close to the source. Would love to have your take on this?

Reply:
hmm, that would only allow the CustomVariableEditor to not be connected. The other two that has this option would still need to be connected. So I think I would prefer to keep it as is to be consistent. If that makes sense?

Reply:
Yes, that makes perfect sense.